src/App.js

```diff
@@ -85,7 +86,7 @@ const App = compose(injectState, withApi)(({ editing, setEditing, state, api })
             forceSelectRole({
               api,
               isLoadingUser,
-              Component: UserProfile,
+              Component: withPageViewTracker(UserProfile),
```

Comment:
view comment @ bottom re: HOC, I think we may be able to avoid the HOC altogether